lib/src/base-styles/_nx-pagination.scss

```diff
@@ -66,21 +71,22 @@
   }
 
   &:active {
-    background-color: var(--nx-swatch-grey-97);
-    color: var(--nx-color-text);
+    background-color: var(--nx-color-pagination-background-active);
+    color: var(--nx-color-text-stark);
     outline: none;
   }
 
   &.selected {
     &, &:hover {
-      background-color: var(--nx-swatch-blue-40);
+      background-color: var(--nx-color-primary-button-background);
       border-color: transparent;
-      color: var(--nx-color-component-background);
+      color: var(--nx-color-pagination-focus-selected);
```

Comment:
So these have the background of a primary button but a different text color?

Reply:
> So these have the background of a primary button but a different text color?

Talked with Jason and now they're following the primary button styles. Check c554154f716c300a3a9efce306a4a27379ed355d